.github/workflows/test-all-versions.yml

```diff
@@ -22,8 +22,6 @@ jobs:
         include:
           - node: "18"
             lerna-extra-args: >-
-              --ignore @opentelemetry/instrumentation-fastify
```

Comment:
This PR is for restify but it also remove ignoring fastify. Is it intentional?

Reply:
yes, I miss this part in my PR in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1568
Do you want to split this?

Reply:
Cool , no need to split